lib/licenses.nix

```diff
@@ -624,6 +624,12 @@ lib.mapAttrs (n: v: v // { shortName = n; }) rec {
     url = http://metadata.ftp-master.debian.org/changelogs/main/d/debianutils/debianutils_4.8.1_copyright;
   };
 
+  sspl = {
+    shortName = "SSPL";
+    fullName = "Server Side Public License";
+    url = https://www.mongodb.com/licensing/server-side-public-license;
+  };
```

Comment:
This lacks a `free = false;`, as the SSPL is not recognized by the OSI and also has been [dropped from Fedora for that reason.](https://www.geekwire.com/2019/mongodbs-licensing-changes-led-red-hat-drop-database-latest-version-server-os/)